components/d2l-all-assessments-list-item.js

```diff
@@ -20,7 +20,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-all-assessments-list-item">
 
 			.assessment-info {
 				display: flex;
-				margin: 10px auto;
+				margin: 10px 10px;
```

Comment:
This would be the same just slightly shorter

```suggestion
				margin: 10px;
```

Reply:
Yes, that sounds good. Changes have been made.